eng/pipelines/templates/steps/install-rex-validation-tool.yml

```diff
@@ -1,4 +1,19 @@
 steps:
+  - pwsh: |
```

Comment:
Should we look at using this DevOps built-in command using `pre-installed` as the source

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/java-tool-installer-v0?view=azure-pipelines#syntax

Reply:
That task is exactly what I was searching for, with no luck. I'm going to update things.